lib/access.js

```diff
@@ -20,6 +20,10 @@ const subcommands = [
 ]
 
 class Access extends BaseCommand {
+  static get description () {
+    return 'Set access level on published packages'
```

Comment:
These all now should match the descriptions in the docs for all the commands, which means we are one step away from being able to programmatically generate everything before the longhand "Description" sections of those docs.